core/nodepool/config/templates/cloud-config-worker

```diff
@@ -481,7 +481,7 @@ coreos:
         --container-runtime={{.ContainerRuntime}} \
         --rkt-path=/usr/bin/rkt \
         --rkt-stage1-image=coreos.com/rkt/stage1-coreos \
-        --node-labels=kubernetes.io/role=node{{if .NodeLabels.Enabled}},{{.NodeLabels.String}}{{end}} \
+        --node-labels=kubernetes.io/role={{ toLabel .NodePoolName }}{{if .NodeLabels.Enabled}},{{.NodeLabels.String}}{{end}} \
```

Comment:
probably it is worth keeping `node-role.kubernetes.io/node=` label to be able to select all worker nodes should you need to

Reply:
I will add that label in so it could be used in the future.